microcosm_pubsub/chain/statements/call.py

```diff
@@ -0,0 +1,51 @@
+"""
+Syntactic sugar using existing primitives defined
+
+"""
+from microcosm_pubsub.chain import Chain
+from microcosm_pubsub.chain.context import ScopedSafeContext
+
+
+class LocalCallWrapper:
+    """
+    Syntactic sugar making it easy to call the same function multiple times in a chain
+    Usage:
+    Chain(
+       ...,
+       call(a_function).with_args(a_function_arg="an_existing_context_key").as_("name_of_the_result_in_context"),
+       call(a_function).with_args(a_function_arg="some_other_key").as_("another_name"),
+    )
+
+    """
+    def __init__(self, func):
+        self.func = func
+        self.chain = Chain(func)
+        self.kwargs_mapping = dict()
+        self.result_name = None
+        self.local_context = None
+
+    def with_args(self, **kwargs):
+        self.kwargs_mapping = kwargs
+        return self
+
+    def as_(self, name):
+        self.result_name = name
+        return self
+
+    def _build_local_kwargs(self, parent_context):
+        return {
```

Comment:
Please use context/safe context from https://github.com/globality-corp/microcosm-pubsub/blob/develop/microcosm_pubsub/chain/context.py

Reply:
@pelegdvir that's what I'm doing (see `__call__` below) - but I need the logic to pass the right kwargs to the scoped context.

Reply:
Right!